config/open-telemetry.php

```diff
@@ -94,4 +94,16 @@
      * This class generates ids for both traces and spans.
      */
     'id_generator' => Spatie\OpenTelemetry\Support\IdGenerator::class,
+
+    /*
+     * Injectors configure, how span will be injected into other structures
```

Comment:
Let's first try to have the simplest implementation possible, without this config. Could you adjust this PR so that it just adds the macro and the tests to make sure that it works?